doc/userguide/configuration/suricata-yaml.rst

```diff
@@ -1686,6 +1686,37 @@ the limits are exceeded, and an event will be raised.
 `max-write-queue-size` and `max-write-queue-cnt` are as the READ variants,
 but then for WRITEs.
 
+Cache limits
+^^^^^^^^^^^^
+
+The SMB parser uses several per flow caches to track data between different records
+and transactions.
```

Comment:
Add a sentence to explain that these caches have a limit to understand the list below with the numbers

Reply:
Added something.